CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 1.0.4 under development
 
-- no changes in this release.
+- Chg #121: Add `network-utilities` dependency and use it instead of `validator` for `IpFilter` (@arogachev)
+- Chg #121: Add support for `validator` of version 2.0, mark it as deprecated (@arogachev)
```

Comment:
```suggestion
- Enh #121: Add `network-utilities` dependency and use it instead of `validator` for `IpFilter` (@arogachev)
- Enh #121: Add support for `validator` of version 2.0, mark it as deprecated (@arogachev)
```